packages/ui/src/theme/css/component/sliderField.scss

```diff
@@ -130,19 +147,22 @@
   &:focus {
     box-shadow: var(--amplify-components-sliderfield-thumb-focus-box-shadow);
   }
-  
+
   &[data-disabled] {
```

Comment:
Why not flatten `disabled`?

Reply:
I think it's because we can't access the underlying Radix component to add our own flattened classes.